amlb/frameworks/definitions.py

```diff
@@ -28,19 +32,24 @@ def _load_and_merge_framework_definitions(frameworks_file: Union[str, List[str]]
     if not isinstance(frameworks_file, list):
```

Comment:
Maybe we should remove the ` log.info("Loading frameworks definitions from %s.", frameworks_file)` line above this? It's rather confusing because it actually loads all definitions across tags. Also is there a reason we load all definitions, is there ever a fall back from e.g. `2020Q2` to the default tag?

Reply:
Only reason is to minimize changes. 
But loading everything on startup you also brings some benefit as all definition files are loaded and validated on startup: you don't have to test them all, one after another...

We may want to change this approach when we'll have too many 202xQn definition files, but for now, it allows to keep API as it was, without to change the resources logic, and this should be very useful for https://github.com/openml/automlbenchmark/issues/169